### PR TITLE
Add policy control for URL import settings

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/72911

This PR adds an option field to control whether an event presentation settings should be imported or not alongside its content when importing from the Other URL source.

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Each line in an array declaration must end in a comma | [Abstract.php:29](https://github.com/moderntribe/the-events-calendar/blob/2e7b2fb767ad257906e92eb6ff91e1e03bafef50/src/Tribe/Aggregator/Record/Abstract.php#L29)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 1]
```diff
@@ -1038,8 +1045,9 @@ public function insert_posts( $items = array() ) {
 		//if we have no non recurring events the message may be different
 		$non_recurring = false;
 
-		$show_map_setting = Tribe__Events__Aggregator__Settings::instance()->default_map( $this->meta['origin'] );
-		$update_authority_setting = Tribe__Events__Aggregator__Settings::instance()->default_update_authority( $this->meta['origin'] );
+		$origin                   = $this->meta['origin'];
+		$show_map_setting         = 'yes' === Tribe__Events__Aggregator__Settings::instance()->default_map( $origin );
```
Comment (bordoni) [inline src/Tribe/Aggregator/Record/Abstract.php]: Use `tribe_is_truthy`.

Comment (lucatume) [inline src/Tribe/Aggregator/Record/Abstract.php]: Had no idea such a function existed...

**Comment by lucatume** [comment]
and there it is

**Comment by bordoni** [comment]
Merge away.

**Pull request merged**